rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```diff
@@ -73,14 +73,32 @@ type = "eql"
 
 query = '''
 iam where winlog.api:"wineventlog" and event.action == "added-member-to-group" and
-  group.name : ("Admin*",
-                "Local Administrators",
-                "Domain Admins",
-                "Enterprise Admins",
-                "Backup Admins",
-                "Schema Admins",
-                "DnsAdmins",
-                "Exchange Organization Administrators")
+(
+    (
+        group.name : (
```

Comment:
I see that this is mainly admin focused, but here are some additional considerations maybe?

```
"Account Operators",
"Server Operators",
"Print Operators",
"Remote Desktop Users",
"Power Users",
"Hyper-V Administrators",
"Remote Management Users",
"Cryptographic Operators",
"Network Configuration Operators",
```

```
"S-1-5-32-548",
"S-1-5-32-549",
"S-1-5-32-550",
"S-1-5-32-555",
"S-1-5-32-547",
"S-1-5-32-556",
"S-1-5-32-569",
"S-1-5-32-580",
```

Reply:
Just added a few more that were missing